server/serverConstants.js

```diff
@@ -99,6 +99,10 @@ E.DEFAULT_INDEX_PAGE = { envVar: 'DEFAULT_INDEX_PAGE', jsonPath: 'defaultPageInd
 
 E.SHOW_TOS = { envVar: 'SHOW_TOS', jsonPath: 'showTos', defaultValue: false, parser: parseBool };
 
+E.SUBSCRIBER_RECEIVED_FRAMERATE = { envVar: 'SUBSCRIBER_RECEIVED_FRAMERATE', jsonPath: 'OpenTok.subscriberReceivedFramerate', defaultValue: 15 };
+
+E.PUBLISHER_RESOLUTION = { envVar: 'PUBLISHER_RESOLUTION', jsonPath: 'OpenTok.publisherResolution', defaultValue: '640x480' };
```

Comment:
Change the default to "1280x720", and we will set this in the environment variable.